Sources/Controllers/TargetMenu/GPX/OAGPXListViewController.mm

```diff
@@ -1202,18 +1203,22 @@ - (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(
         }
         else if ([menuCellType isEqualToString:[OAIconTextTableViewCell getCellIdentifier]])
```

Comment:
This is very wrong. You need to change the menuCellType in the data to OARightIconTableViewCell.